src/app/components/character-card/character-card.component.scss

```diff
@@ -0,0 +1,95 @@
+@import '/src/scss/colors';
+
+.card {
+  align-items: flex-start;
+  border: 1px solid $disabled;
+  border-radius: 15px;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  height: 350px;
+  justify-content: space-around;
+  max-width: 250px;
+  padding: 8px 15px;
+  position: relative;
+  width: 100%;
+  
+  .picture {
+    border-radius: 10px;
+    height: 180px;
+    object-fit: cover;
+    width: 100%;
+  }
+
+  .title {
+    color: $wolox-blue;
+    font-size: 20px;
+    font-weight: bold;
+  }
+  
+  .tag {
+    border-radius: 5px;
+    color: white;
+    font-weight: bold;
+    padding: 3px 5px;
+  }
+
+  .alive {
+    background-color: $pear;
+  }
+  
+  .dead {
+    background-color: $crimson;
+  }
+  
+  .unknown {
+    background-color: $disabled;
+  }
+
+  .detail {
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    width: 80%;
+
+    .bold {
+      font-weight: bold;
+    }    
+  }
+
+  .select {
+    appearance: none; 
+    border: 4px solid $lightgray;
+    border-radius: 50%;
+    bottom: 20px;
+    cursor: pointer;
+    height: 35px;
+    position: absolute;
```

Comment:
Why was this style added?

Reply:
I decided to convert the checkbox in an absolute elment in order to position it in the specific spot.